gulpfile.js/tasks/comp-lib.js

```diff
@@ -6,14 +6,21 @@ var gulp = require('gulp')
 var exec = require('child_process').exec
 var config = require('../config')
 var compLibConfig = require('../../component-lib.json')
+var os = require('os')
 
 gulp.task('clean-comp-lib', function (cb) {
   del(compLibConfig.destination, cb)
 })
 
 gulp.task('component-library', ['clean-comp-lib', 'sass', 'images', 'browserify'], function (cb) {
   var env = global.runmode
-  var genCompLib = './node_modules/.bin/kss-node --config component-lib.json'
+  var genCompLib
+
+  if (os.platform() === 'win32') {
```

Comment:
This doesn't need to be branched. [`path.join()`](https://nodejs.org/docs/latest/api/path.html#path_path_join_paths) or [`path.resolve()`](https://nodejs.org/docs/latest/api/path.html#path_path_resolve_paths) will [join their args with the right separators for each platform](https://nodejs.org/docs/latest/api/path.html#path_windows_vs_posix).